zun_kata.md

```diff
@@ -0,0 +1,126 @@
+# OpenStack Zun DevStack working with Kata Containers
+## Introduction
+
+This guide describes how to get Kata Containers to work with OpenStack Zun
+using DevStack on Ubuntu 16.04. Running DevStack with this guide will setup
+Docker and Clear Containers 2.0, which you replace with Kata Containers.
+Currently, the instructions are based on the following links:
+
+- https://docs.openstack.org/zun/latest/contributor/quickstart.html
+
+- https://docs.openstack.org/zun/latest/admin/clear-containers.html
+
+- https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md
+
+
+## Install git to use with DevStack
+
+```
+sudo apt install git
+```
+
+## Setup OpenStack DevStack
+The following commands will sync DevStack from github, create your
```

Comment:
Replaced "github" with "GitHub": The following commands will sync DevStack from GitHub, create your